setup/plugins/checkRequirements/CheckDbWellFormed.civi-setup.php

```diff
@@ -13,7 +13,12 @@
   ->addListener('civi.setup.checkRequirements', function (\Civi\Setup\Event\CheckRequirementsEvent $e) {
     \Civi\Setup::log()->info(sprintf('[%s] Handle %s', basename(__FILE__), 'checkRequirements'));
 
-    $dbFields = array('db', 'cmsDb');
+    $dbFields = ['db'];
+
+    if ($e->getModel()->cms !== 'Standalone') {
```

Comment:
Test failed because `CIVICRM_UF` was not set. The `$model->cms` is the pre-installation equivalent. Pushed a tweak. for that.

Reply:
@totten thanks :)